framework/src/partitioner/PetscExternalPartitioner.C

```diff
@@ -14,6 +14,7 @@
 
 #include "libmesh/mesh_tools.h"
 #include "libmesh/linear_partitioner.h"
+#include "libmesh/sfc_partitioner.h"
```

Comment:
This looks like an atavism from intermediate debugging steps, not a header we need now.